### PR TITLE
User interface for review code

### PR DESCRIPTION
**Pull request by digomes87**

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,124 @@
-#!/usr/bin/env ruby
-puts "Hello world!"
-#new branch
+##!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require_relative '../lib/game.rb'
+require_relative '../lib/players.rb'
+require_relative '../lib/board.rb'
```
Comment (maelfosso) [inline bin/main.rb]: 1. How is it possible while the **lib** folder is empty ?
2. For this **milestone** you should only work into **main.rb** file

File: bin/main.rb [hunk 2]
```diff
@@ -1,3 +1,124 @@
...
+
+    puts 'ERRORRRR! It should be X or O,  *_*'
+    element = gets.chomp.upcase!
+  end
+end
+
+def info_players
+  puts "\nOk! Player 1 your name: "
+  name = gets.chomp
+  player1 = Player.new(name_valid?(name))
```
Comment (maelfosso) [inline bin/main.rb]: Where is this class ?
In that milestone, you're not supposed to have such a class. 
You don't have to think about classes

File: bin/main.rb [hunk 3]
```diff
@@ -1,3 +1,124 @@
...
+  player2.element = player1.element == 'X' ? 'O' : 'X'
+  [player1, player2]
+end
+
+players = info_players
+player1 = players[0]
+player2 = players[1]
+
+loop do
+  board = Board.new
```
Comment (maelfosso) [inline bin/main.rb]: Same remark as **Player**
Don't think about classes in that milestone

File: bin/main.rb (hunk 1 above)
Comment (digomes87) [reply bin/main.rb]: this happing because I working thinking in another step.

Now I'm remove this lines

File: bin/main.rb (hunk 2 above)
Comment (digomes87) [reply bin/main.rb]: I should remove this class ?

File: bin/main.rb (hunk 3 above)
Comment (digomes87) [reply bin/main.rb]: and this ?

**Pull request merged**